### PR TITLE
VZ-5890 Minor changes to VZ status command

### PR DESCRIPTION
**Pull request by mgianatagh**
# Description

- Configure the stdin setting for a cmd
- Change error message for VZ not found to quote the name of the object to handle case of no name passed.
- Disable printing the help text any time a command fails

Fixes VZ-5890

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tools/vz/cmd/status/status_test.go [hunk 1]
```diff
@@ -105,15 +105,15 @@ func TestStatusCmd(t *testing.T) {
 	statusCmd.SetArgs([]string{fmt.Sprintf("--%s", nameFlag), name, fmt.Sprintf("--%s", namespaceFlag), "default"})
 	err = statusCmd.Execute()
 	assert.Error(t, err)
-	assert.True(t, strings.Contains(errBuf.String(), "Failed to find Verrazzano with name verrazzano in namespace default"))
+	assert.True(t, strings.Contains(errBuf.String(), "Failed to find Verrazzano with name \"verrazzano\" in namespace default"))
```
Comment (kminder) [inline tools/vz/cmd/status/status_test.go:108]: Should the namespace be quoted also?

File: tools/vz/cmd/status/status.go [hunk 2]
```diff
@@ -170,7 +170,7 @@ func runCmdStatus(cmd *cobra.Command, args []string, vzHelper helpers.VZHelper)
 	vz := vzapi.Verrazzano{}
 	err = client.Get(context.TODO(), types.NamespacedName{Namespace: namespace, Name: name}, &vz)
 	if err != nil {
-		return fmt.Errorf("Failed to find Verrazzano with name %s in namespace %s: %v", name, namespace, err.Error())
+		return fmt.Errorf("Failed to find Verrazzano with name %q in namespace %s: %v", name, namespace, err.Error())
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply tools/vz/cmd/status/status.go:173]: If the user explicitly sets namespace to "" then they would see this:

vz status --name v8o --namespace ""
Error: Failed to find Verrazzano with name "v8o" in namespace : an empty namespace may not be set when a resource name is provided

I should quote the namespace too.  Will push that change.

File: tools/vz/cmd/status/status_test.go (hunk 1 above)
Comment (mgianatagh) [reply tools/vz/cmd/status/status_test.go]: yes it should, I just pushed that change

**Review by kminder — APPROVED** [review]
Approved.

**Review by bobdonat — APPROVED** [review]

**Pull request merged**